app/routes/goal_routes.py

```diff
@@ -0,0 +1,118 @@
+
+from flask import Blueprint, jsonify, request, make_response
+from app import db 
+from app.models.goal import Goal
+from app.models.task import Task
+from .routes_helper import validate
+
+
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return make_response({"details": "Invalid data"},400)
```

Comment:
It's good that you're doing validation, but you should also include details to the user about what went wrong.